vm/src/builtins/bytearray.rs

```diff
@@ -123,31 +124,37 @@ impl PyByteArray {
         self.inner.write()
     }
 
+    /// Return repr(self).
     #[pymethod(magic)]
     fn repr(&self) -> String {
         self.inner().repr("bytearray(", ")")
     }
 
+    /// B.__alloc__() -> int\n\nReturn the number of bytes actually allocated.
```

Comment:
It shouldnt have signature. I didnt test it but i am pretty sure signature will be double stacked if it is manually input

Reply:
yeah, the latest changes will be the same as your suggestions @fanninpm , i didnt push them yet. Also, clippy drops
`///` when there is no character in the line, i i think it will be in the end something like:
```rust
    /// B.__alloc__() -> int \n
    /// \n
    /// Return the number of bytes actually allocated.
```

Reply:
> Also, clippy drops /// when there is no character in the line

Pretty sure clippy doesn't do that since that would clearly violate the intent of the programmer writing the docstring (to have an empty new line when docs are rendered). 

Currently, docstrings should literally just be copied and pasted, e.g for `list.pop`:

```python
print(list.pop.__doc__)
Remove and return item at index (default last).

Raises IndexError if list is empty or index is out of range.
```

should be added as:

```rust
/// Remove and return item at index (default last).
///
/// Raises IndexError if list is empty or index is out of range.
fn pop(...)
```

and that should do the trick!

Reply:
thanks, will implement the suggestions :)